expyfun/stimuli/_tracker.py

```diff
@@ -93,6 +94,10 @@ class TrackerUD(object):
         The minimum value that the tracker level (``x``) is allowed to take.
     x_max : float
         The maximum value that the tracker level (``x``) is allowed to take.
+    repeat_limit : str
+        How to treat trials that hit either x_min or x_max. ``reversals`` will
+        count these trials as reversals while staying at the same level.
+        ``ignore`` does not automatically count these trials as reversals.
```

Comment:
For some reason I thought we didn't want the subject to touch a ceiling or floor at all (probably based on the way I was using it). But what you're saying makes sense. I will make adjustments.